bigquery/google/cloud/bigquery/external_config.py

```diff
@@ -29,6 +29,33 @@
 from google.cloud.bigquery.schema import SchemaField
 
 
+class ExternalSourceFormat(object):
+    """The format for external data files..
+
+    Note that the set of allowed values for external data sources is different
+    than the set used for loading data (see
+    :class:`~google.cloud.bigquery.job.ExternalSourceFormat)`.
```

Comment:
I'm actually a bit surprised that the external sources for creating tables would be different for those from queries.

@shollyman Is this this expected?

Reply:
@tswast FTR:

- https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.load.sourceFormat
- https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#externalDataConfiguration.sourceFormat